### DISCUSSION TITLE
[Solved] Badges error on profile

### DISCUSSION THREAD

**Discussion by ghost** [community/community#101403] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Hey guys! I've been having a problem since I created my account (recently).

The badges don't appear on my github profile on the web (I only have one at the moment), but they appear on the mobile app.

I've already tried a few things like changing browsers and also clearing browsing cookies, but they simply don't appear!

![image](https://github.com/community/community/assets/153748817/80c91f1f-416b-4679-97a4-2e119a3e25c1)

Não existe essa opção.

oh, and I also activated the option to show badges in my profile settings

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by brenoepics** [comment]
Hey @henriquenandrade , try to open [GitHub settings -> public profile](https://github.com/settings/profile), search for this and check it
![image](https://github.com/community/community/assets/59066707/37911d0f-fc6e-48aa-80e6-3dd3c41910e6)
then click on update button.

**Reply by ghost** [reply]
Thanks for the answer. Yes, I have that option checked.

![image](https://github.com/community/community/assets/153748817/78e1b491-587d-4f7a-8b40-76d86ed63007)

**Reply by ghost** [reply]
it still doesn't appear

**Comment by Chupriarti** [comment]
There is a button to hide a certain achievement. If all achievements are hidden like this, then the block with them is not displayed
![achiv-problem](https://github.com/community/community/assets/47197404/07cfe690-4660-4a9b-a5e7-2c7392f5e41a)

**Reply by ghost** [reply]
I understand, but I didn't manage to change the option and also when I click on edit profile the badges don't appear

**Comment by ghost** [comment]
My badge dont apper too

**Reply by ghost** [reply]
Hello everything is fine?
Strange, they appear when I access your profile

**Reply by ghost** [reply]
Did you manage to solve this?

**Reply by ghost** [reply]
Try this [Tutorial](https://github.com/Thinkright20/Profile-Badges), it may help.

**Comment by brenoepics** [comment, marked as answer]
![image](https://github.com/community/community/assets/59066707/b7fa6e9a-fe40-40f7-8d55-bae769e6df94)

@henriquenandrade I made a co-authored pull request using your username, it should be working now

**Reply by ghost** [reply]
Thank you very much! You saved the day haha

If you need anything I'm here ^^

“My hero doesn’t wear a cape, he uses GH”

**Reply by OlegBayrak** [reply]
@brenoepics I have the same issue, there's no achievements displayed on my profile - though I made all the steps above.
Can you please help me?

**Reply by ghost** [reply]
I believe he or someone will have to do the same process of making a pull request mentioning you as a co-author.
I'll wait for @brenoepics to respond so we can understand what it could be.

**Reply by brenoepics** [reply]
> @brenoepics I have the same issue, there's no achievements displayed on my profile - though I made all the steps above. Can you please help me?

Actually it was because you didn't have any achievements yet, I made a commit with you as co-author and it appeared.

![image](https://github.com/user-attachments/assets/2358708f-cfa3-487f-a3d6-b58c91a98195)

**Reply by OlegBayrak** [reply]
Thanks - now I can see it!
